code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm

```diff
@@ -217,3 +215,35 @@
 		. += "Lifetime Hugs: [total_facehugs] / [next_facehug_goal]"
 	else
 		. += "Lifetime Hugs: [total_facehugs]"
+
+
+/datum/xeno_mutator/watcher
+	name = "STRAIN: Facehugger - Watcher"
+	description = "You lose your ability to hide in exchange to see further and the ability to no longer take damage outside of weeds. This enables you to stalk your host from a distance and wait for the perfect oppertunity to strike."
+	flavor_description = "No need to hide when you can see the danger."
+	individual_only = TRUE
+	caste_whitelist = list(XENO_CASTE_FACEHUGGER)
+	mutator_actions_to_remove = list(
+		/datum/action/xeno_action/onclick/xenohide,
+	)
+	mutator_actions_to_add = list(
+		/datum/action/xeno_action/onclick/toggle_long_range/runner,
+	)
+
+	cost = 1
+
+	keystone = TRUE
+
+/datum/xeno_mutator/watcher/apply_mutator(datum/mutator_set/individual_mutators/mutator_set)
+	. = ..()
+	if(!.)
+		return
+
+	var/mob/living/carbon/xenomorph/facehugger/facehugger = mutator_set.xeno
+
+	facehugger.viewsize = 10
+	facehugger.layer = 4 // Mob layer is 4 but we don't have a define for it? It's commented out in the layers.dm file
```

Comment:
that's because MOB_LAYER is a dm define, you can just use it

Reply:
done